### PR TITLE
Fixed typo in pr_workflow.rst

### PR DESCRIPTION
**Pull request by ramadm**

### CODE REVIEW

File: contributing/workflow/pr_workflow.rst [hunk 1]
```diff
@@ -436,7 +436,7 @@ The interactive rebase
 
 If you didn't follow the above steps closely to *amend* changes into a commit
 instead of creating fixup commits, or if you authored your changes without being
-aware of our workflow and Git usage tips, reviewers might request of your to
+aware of our workflow and Git usage tips, reviewers might request of you to
```
Comment (skyace65) [inline contributing/workflow/pr_workflow.rst:439]:
```suggestion
aware of our workflow and Git usage tips, reviewers might request you to
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
I'd get rid of the "of" in that sentence to make it sound better.

**Comment by ramadm** [comment]
Hi, I committed your change. I see that the docs suggested to keep changes to a single commit when possible, but I'm not sure if it's possible to do that via GitHub's online editor. If you want me to delete and re-submit the PR as a single commit let me know.

**Comment by AThousandShips** [comment]
It's okay in this case, but in the future please use the computer setup, just editing from the web interface is very limited 🙂

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
I've done a squash and merge for this PR so it's only 1 commit, but in the future I'd highly recommend getting setup with a proper editor on your computer for using Git. If you don't know anything about Git we have a guide in the Godot docs [here](https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html).

Also congrats on your first merged PR!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2.

**Pull request merged**